apps/files/lib/Service/UserConfig.php

```diff
@@ -146,6 +149,10 @@ public function getConfigs(): array {
 			return $value;
 		}, $this->getAllowedConfigKeys());
 
-		return array_combine($this->getAllowedConfigKeys(), $userConfigs);
+		$configs = array_map(function (string $key) use ($userConfigs) {
+			return $this->appConfig->getAppValueBool($key, $userConfigs[$key] ?? $this->getDefaultConfigValue($key));
```

Comment:
`$userConfig` is not key => value array :( - it is array of values `[ true, false, true, ...]`

please check the pr for fix https://github.com/IONOS-Productivity/nc-server/pull/64

Reply:
review applied :heavy_check_mark: